newrelic-security-agent/src/main/java/com/newrelic/agent/security/instrumentator/dispatcher/Dispatcher.java

```diff
@@ -226,6 +233,19 @@ public Object call() throws Exception {
         return null;
     }
 
+    private JavaAgentEventBean prepareCachingDataStoreEvent(JavaAgentEventBean eventBean, RedisOperation redisOperation) {
+        JSONArray params = new JSONArray();
+        for (Object data : redisOperation.getArguments()) {
+            params.add(data);
+        }
+        JSONObject command = new JSONObject();
+        command.put(REDIS_MODE, redisOperation.getMode());
+        command.put(REDIS_ARGUMENTS, params);
+        command.put(REDIS_TYPE, redisOperation.getType());
+        eventBean.setParameters(params);
```

Comment:
Here set `command` object instead of `params` to `eventBean`

Reply:
Fixed